waddrmgr/db.go

```diff
@@ -1622,6 +1623,11 @@ func fetchAddressByHash(ns walletdb.ReadBucket, scope *KeyScope,
 		return deserializeScriptAddress(row)
 	case adtWitnessScript:
 		return deserializeWitnessScriptAddress(row)
+	case adtTaprootScript:
```

Comment:
Maybe I'm missing something, but iirc we already did the base keyspend address integration here?

Reply:
Not 100% sure what you mean. We do add the `TaprootPubKey` address type in this PR. But because that is just a different representation of a private/public key pair, there doesn't need to be any specific database code for it. Same as there is no specific p2wkh serialization/de-serialization code here, only p2wsh.

Reply:
I was confused, I think you've clarified things lower re the two diff taproot addr types.